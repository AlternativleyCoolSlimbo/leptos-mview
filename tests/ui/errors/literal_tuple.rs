use leptos_mview::view;

fn main() {
    view! {
        span attr=(a, b);
    };
}
