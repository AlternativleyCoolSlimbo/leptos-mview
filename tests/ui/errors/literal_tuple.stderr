error: unexpected token
 --> tests/ui/errors/literal_tuple.rs:5:21
  |
5 |         span attr=(a, b);
  |                     ^
  |
  = note: if trying to pass a tuple, wrap it in additional braces or parens. e.g. ((a, b)) or {(a, b)}.
